tests/cli/test_train_model.py

```diff
@@ -26,6 +27,33 @@ def test_train(monkeypatch, tmp_path, output):
     assert Path(output).is_file()
 
 
+@pytest.mark.parametrize("output", [None, "mymodel.pt"])
+def test_train_explicit_validation_test(monkeypatch, tmp_path, output):
```

Comment:
Thanks for adding the test. We should also test a combination of these. Like both as a float each one as a float and both as files.

Reply:
Done!

Reply:
Hmm I don't know if like that we add three more files here. Maybe keep the explicit and load this one, overwrite them and save them to the custom directory with the float options.